transports/uds/src/lib.rs

```diff
@@ -36,6 +36,7 @@
     not(target_os = "emscripten"),
     any(feature = "tokio", feature = "async-std")
 ))]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 #![cfg_attr(docsrs, doc(cfg(all(unix, not(target_os = "emscripten")))))]
```

Comment:
That is the last occurrence of `doc(cfg` that ripgrep finds on my machine!

```suggestion
```

Reply:
Oh my bad. I used `#\[cfg_attr\(docsrs` expression. Missing out on global attributes.